src/Widgets/ImageWidget/ImageWidgetClass.js

```diff
@@ -6,6 +6,19 @@ const ImageWidget = Scrivito.provideWidgetClass("ImageWidget", {
     alignment: ["enum", { values: ["left", "center", "right"] }],
     alternativeText: "string",
     link: "link",
+    animation: [
+      "enum",
+      {
+        values: [
+          "",
```

Comment:
Enum values can be deselected and therefor have no animation. We do not need an addition `""` (aka "none")  state for that.